index.html

```diff
@@ -0,0 +1,265 @@
+          <i class="material-icons sprtr"></i>
+          <span class="deets">2015</span>
+        </div>
+        </div>
+        <p class="prt-txt2">
+        Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.
+        </p>
+        <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">Ruby on rails</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+        </ul>
+        <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+        </div>
+      </div>
+      <img src="./elements/prtflwrks2.png" alt="Multi-Post-">
+    </div>
+    <div class="port">
+      <img src="./elements/prtflwrks3.png" alt="Tonic">
+      <div class="prt-txt">
+        <div class="prt-txt1">
+          <h2 class="prt-ttl">Facebook 360</h2>
+          <div class="frm2">
+          <span class="clnt">FACEBOOK</span>
+          <i class="material-icons sprtr"></i>
+          <span class="deets">Full Stack Dev</span>
+          <i class="material-icons sprtr"></i>
+          <span class="deets">2015</span>
+          </div>
+        </div>
+        <p class="prt-txt2">
+          Exploring the future of media in Facebook's first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.
+        </p>
+        <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">Ruby on rails</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+        </ul>
+        <div>
+          <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+          </button>
+        </div>
+      </div>
+    </div>
+    <div class="port">
+      <div class="prt-txt">
+        <div class="prt-txt1">
+        <h2 class="prt-ttl">Uber Navigation</h2>
+        <div class="frm2">
+          <span class="clnt">Uber</span>
+          <i class="material-icons sprtr"></i>
+          <span class="deets">Lead Developer</span>
+          <i class="material-icons sprtr"></i>
+          <span class="deets">2018</span>
+        </div>
+        </div>
+        <p class="prt-txt2">
+          A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.
+        </p>
+        <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">Ruby on rails</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+        </ul>
+        <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+        </div>
+      </div>
+      <img src="./elements/prtflwrks4.png" alt="Multi-Post-">
+    </div>
+  </section>
+
+  <!-- About Section -->
+    <section id="about">
+    <div class="abt-txt1">
+      <i class="abt-prmrytxt1">
+        <h1>
+      About <br> Myself
+        </h1>
+      </i>
```

Comment:
- [ ] When you finally adjust the screen resolution based on the changes I suggested prior to this comment, please also add margins in between the text of the `About` and `Myself` `h1` text, just as it's specified on the figma design. The reason for this stems from keeping a good UI that will be as less flawed as possible to give a great view to the user.

# Your design 
![image](https://user-images.githubusercontent.com/50990925/213594781-403feeaa-066a-4bf5-ae15-a90f27197bc2.png)

# Figma
![image](https://user-images.githubusercontent.com/50990925/213594674-17cb97e7-e091-431d-9497-4dfb922022b1.png)